src/main/java/goodgenerator/blocks/tileEntity/LargeEssentiaSmeltery.java

```diff
@@ -469,7 +469,7 @@ public boolean onRunningTick(ItemStack aStack) {
                     default:
                         y += 1;
                 }
-                if (WORLD.getBlock(x, y, z) instanceof BlockAir) generateFluxGas(WORLD, x, y, z);
```

Comment:
Why this was removed? @DianeXD

Reply:
Players place blocks to get around this. Since this is magical "pollution", there is no need to create another pr